cppcache/src/ClientProxyMembershipID.cpp

```diff
@@ -51,7 +51,7 @@ ClientProxyMembershipID::ClientProxyMembershipID()
       m_hostAddrLocalMem(false),
       m_vmViewId(0) {}
 
-ClientProxyMembershipID::~ClientProxyMembershipID() noexcept {
+ClientProxyMembershipID::~ClientProxyMembershipID() {
```

Comment:
Why remove the `noexcept`?

Reply:
Per C++11 spec, all destructors and only destructors are implicitly noexcept(true). that makes this redundant. There are edge cases where you can willfully break things but I'm not going to quote every nuance of the spec.

Reply:
https://en.cppreference.com/w/cpp/language/noexcept_spec
> default constructors, copy constructors, move constructors that are implicitly-declared or defaulte on their first declaration unless
The keyword being "default". In this case these destructors are not default. Can you point me at the spec that says non-default are also implicitly `noexcept`?